src/database/seeders/20190906184341-Users.js

```diff
@@ -1,27 +1,128 @@
+import bcrypt from 'bcryptjs';
 import { config } from 'dotenv';
-import Utils from '../../utils';
 
 config();
 
 module.exports = {
-  up: (queryInterface) => queryInterface.bulkInsert(
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert(
     'Users',
     [
+      {
+        firstName: 'Abel',
+        lastName: 'Damina',
+        email: 'abel@gmail.com',
+        password: bcrypt.hashSync(process.env.SECRET, bcrypt.genSaltSync(8)),
+        phoneNumber: 12345678,
+        rememberInfo: false,
+        gender: 'male',
+        birthdate: '01/02/1986',
+        preferredLanguage: 'French',
+        preferredCurrency: 'Naira',
+        address: 'no 3 Ilupeju road, Ikeja, Lagos state.',
+        department: 'IT Deparment',
+        lineManager: 1,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        firstName: 'Dafe',
+        lastName: 'Anna',
+        email: 'dafeanna@gmail.com',
+        password: bcrypt.hashSync(process.env.SECRET, bcrypt.genSaltSync(8)),
+        phoneNumber: 12345678,
+        rememberInfo: false,
+        gender: 'male',
+        birthdate: '01/02/1986',
+        preferredLanguage: 'French',
+        preferredCurrency: 'Naira',
+        address: 'no 3 Ilupeju road, Ikeja, Lagos state.',
+        department: 'IT Deparment',
+        lineManager: 1,
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
       {
         firstName: 'Wetravel',
         lastName: 'user1',
         email: 'wetravel.user1@gmail.com',
-        password: Utils.hashPassword(process.env.SECRET),
+        password: bcrypt.hashSync(process.env.SECRET, bcrypt.genSaltSync(8)),
```

Comment:
Why are you calling bcrypt here, there's already a utility function for hashing passwords which you removed. I don't think that's necessary.

Reply:
when running migrations and seeders the system does not see the util function even though it's imported, that's why I made it so.

Reply:
are you sure, we've been running migrations and it's been working, I think you should test this out on the develop branch.

Reply:
OK. I will test it again.

Reply:
I could only see the chidinma email when I needed them

Reply:
there are more emails. check the seeders file, look up a_userSpec.js you'll see I used some there.

Reply:
@victor-nach done that it's working now